modules/account-lib/src/coin/dot/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { BaseUtils } from '../baseCoin';
+import { NotImplementedError } from '../baseCoin/errors';
+
```

Comment:
Should we derive accounts with the platform? @argjv 
https://polkadot.js.org/docs/ui-keyring/start/accounts/#non-mnemonic-seeds-and-derivation

Reply:
Derivation should be done in the KeyPair, but we don't really need it for DOT